runtime/lua/vim/lsp/util.lua

```diff
@@ -1622,20 +1622,24 @@ function M.open_floating_preview(contents, syntax, opts)
       api.nvim_win_set_var(floating_winnr, opts.focus_id, bufnr)
     end
     api.nvim_buf_set_var(bufnr, 'lsp_floating_preview', floating_winnr)
+    api.nvim_win_set_var(floating_winnr, 'lsp_floating_bufnr', bufnr)
   end
 
-  local augroup_name = ('nvim.closing_floating_preview_%d'):format(floating_winnr)
-  local ok =
-    pcall(api.nvim_get_autocmds, { group = augroup_name, pattern = tostring(floating_winnr) })
-  if not ok then
+  local augroup_name = 'nvim.closing_floating_preview'
+  if vim.fn.exists(('#%s'):format(augroup_name)) == 0 then
```

Comment:
is this check still needed , or can we just always recreate with `group = nvim_create_augroup(augroup_name, { clear = true })` ?

Reply:
But `clear = true` will clear the existing one, so what's the problem? It's worth the code simplification.